### PR TITLE
Tilted titles

### PR DESCRIPTION
**Pull request by jrmartin**
# %%Card name%%
%%Link to the card%%

## Changes
Short changelog

## Related issues
All issues and cards that are connected with the main card

### CODE REVIEW

**Comment by gidili** [comment]
@NeZanyat can you review and test this PR? I will test too once it's merged and deployed.

File: components/TestSuites.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ export default class TestSuites extends React.Component {
         this.styleConfig = {
             classNames: {
                 Table: 'table scidash-table suites-table',
-                TableHeadingCell: 'scidash-table-heading-cell'
+                TableHeadingCell: 'scidash-test-table-heading-cell'
```
Comment (NeZanyat) [inline components/TestSuites.js]: Could you please rename className, to something informative, cause as for now it's not clear what it do

Comment (jrmartin) [reply components/TestSuites.js]: Would this be better?  'scidash-test-suites-table-heading-cell'

That classname is only meant for the table headers in the TestSuites view.

**Comment by gidili** [comment]
Good to merge? @NeZanyat

**Comment by gidili** [comment]
@jrmartin we merged and deployed to dev instance but can't find the tilter headers. Sure you committed everything? 😄

**Comment by jrmartin** [comment]
@gidili Just check, pretty sure it's all there. It shows on the modal dialog when clicking on links in test suites view

https://trello-attachments.s3.amazonaws.com/59bc69b597297c545d33e5ea/5b4724e85dc03b93942a5f84/c49a7a46f411d23a20970c194b80d0e6/Screenshot_from_2018-09-03_09-21-30.png

**Comment by jrmartin** [comment]
@gidili Just double checked with clear cache and notice the lack of tilted titltes

Redid some of the code and opened a new pull request
https://github.com/MetaCell/geppetto-scidash/pull/150

Should do it now.

**Pull request merged**